tests/upgrade_path/test_upgrade_path.py

```diff
@@ -60,11 +60,11 @@ def test_upgrade_path(localhost, duthosts, ptfhost, rand_one_dut_hostname,
             logger.info("Test upgrade path from {} to {}".format(from_image, to_image))
             # Install base image
             logger.info("Installing {}".format(from_image))
-            target_version = install_sonic(duthost, from_image, tbinfo)
+            base_version = install_sonic(duthost, from_image, tbinfo)
```

Comment:
Fixed on the way this wrong variable name: This is the installation of the base version and not target